Apps/Src/Task_CriticalState.c

```diff
@@ -6,6 +7,10 @@
 #include "BSP_UART.h"
 #include "Voltage.h"
 
+/*
+This flag is used to override the undervoltage fault condition of the BPS when we are starting
+the car(if we need to charge). If it is true, then we should be in charging state only.
+*/
 bool AdminOverride = false;
```

Comment:
I think this violates regulation 5.2.E.9: "Firmware or settings will be rendered static and un-modifiable after inspection". It should be safe to remove this and everything that uses it, since we have two separate under voltage cutoffs for charging and discharging. I forgot to remove this when I removed the dashboard override task